src/static/js/routes/careers/working-at-cfpb/index.js

```diff
@@ -0,0 +1,21 @@
+/* ==========================================================================
```

Comment:
I did notice the-bureau needs the same initialization config though. Fixed in https://github.com/cfpb/cfgov-refresh/commit/5ee8753e588140c3eeb0298bdc4983182f806b9f